Rakefile

```diff
@@ -125,23 +131,33 @@ end
 namespace :docs do
   desc 'Generate markdown docs for bolt.yaml'
   task :config_reference do
-    @opts      = stringify_types(Bolt::Config::OPTIONS.slice(*Bolt::Config::BOLT_OPTIONS))
-    @subopts   = Bolt::Config::SUBOPTIONS.dup
-    @inventory = Bolt::Config::INVENTORY_OPTIONS
+    @opts      = Bolt::Config::OPTIONS.slice(*Bolt::Config::BOLT_OPTIONS)
+    @inventory = Bolt::Config::INVENTORY_OPTIONS.dup
 
-    @yaml           = generate_yaml_file(@opts, @subopts)
-    @inventory_yaml = generate_yaml_file(@inventory)
+    # Move sub-options for 'log' option up one level, as they're nested under
+    # 'console' and filepath
+    @opts['log'][:properties] = @opts['log'][:additionalProperties][:properties]
```

Comment:
Should this be additive so that `console` is also documented? It makes sense with the example but is a bit confusing as text - though I'll defer to @hestonhoffman .

Reply:
Ah, I see. So if we leave it like this, `console` is only documented in the description under log and in the examples. The examples do clear things up somewhat, but I think I agree with Lucy that it feels weird to skip a level in the hierarchy. However, we do something similar with `plugin_hooks`, so it's not unprecedented. Maybe if we just clear the language up a bit. Something like:

"A map of configuration for the logfile output. Under `log`, you can configure log options for your `console` and add configuration for individual log files, such as `~/.puppetlabs/bolt/debug.log`."

I think keeping it short and sweet and relying more on the example might be the best way to handle this one.